### ISSUE TITLE
Language tools status stuck spinning forever

### ISSUE THREAD

**Issue by hasen6** [redhat-developer/vscode-java#2266]
My Java project was working up until last night (and has been for months no problem), then I opened vscode today and the status in the language tools at the bottom is constantly spinning and never shows a thumbs up https://github.com/redhat-developer/vscode-java/#troubleshooting

I tried installing 1.63.1 instead but it doesn't change anything.

##### Environment
- Operating System: MacOS 10.13.6
- JDK version: 14
- Visual Studio Code version: 1.63.2
- Java extension version: 16

##### Steps To Reproduce
1. Open vscode.

##### Current Result
Spinning update in status bar at bottom of vscode.

##### Expected Result
Thumbs up in status bar at bottom of vscode.

**Comment by testforstephen** [comment]
Could you pls share the logs? thanks.

**F1 -> Java: Open All Log Files**

**Comment by hasen6** [comment]
Ok this is the log.
[client.log.2021-12-27.txt](https://github.com/redhat-developer/vscode-java/files/7778859/client.log.2021-12-27.txt)

**Comment by hasen6** [comment]
It was my error, never mind.

**Comment by testforstephen** [comment]
Nice to know you have solved the issue. Would you mind sharing the root cause of this error?

**Comment by hasen6** [comment]
> Nice to know you have solved the issue. Would you mind sharing the root cause of this error?

It was something I did the previous day and forgot about. I noticed that the cache files for vscode had gotten really huge, like several gigabytes so I deleted them directly in my library system folder. But when vscode opened the following day it must have created new cache files incorrectly or something, I don't know exactly but it then failed to reach ready status even after opening and re-opening.

Simply closing vscode, deleting all the newly created cache files again and re-opening vscode enabled it to successfully reach ready status.
